src/treemodel/selection.js

```diff
@@ -5,11 +5,15 @@
 
 'use strict';
 
+import Position from './position.js';
+import Range from './range.js';
 import LiveRange from './liverange.js';
 import EmitterMixin from '../emittermixin.js';
 import CKEditorError from '../ckeditorerror.js';
 import utils from '../utils.js';
 
+const storePrefix = 'selection_store:';
```

Comment:
Do we have a naming convention for these? I would say that it should be `selectionStore:`. Or even better – `selection:`.